app/models/pipeline_run.rb

```diff
@@ -26,9 +28,11 @@ class PipelineRun < ApplicationRecord
   PIPELINE_VERSION_FILE = "pipeline_version.txt".freeze
   STATS_JSON_NAME = "stats.json".freeze
   ERCC_OUTPUT_NAME = 'reads_per_gene.star.tab'.freeze
+  AMR_OUTPUT_NAME = 'output__fullgenes__ARGannot_r2__results.txt'.freeze
   TAXID_BYTERANGE_JSON_NAME = 'taxid_locations_combined.json'.freeze
   ASSEMBLY_STATUSFILE = 'job-complete'.freeze
   LOCAL_JSON_PATH = '/app/tmp/results_json'.freeze
+  LOCAL_TXT_PATH = '/app/tmp/results_txt'.freeze
```

Comment:
I know you just wrote something analogous to what already existed (LOCAL_JSON_PATH), but I actually think it would be better to name it something more descriptive, such as `LOCAL_AMR_PATH = '/app/tmp/amr_results'.freeze`.

Reply:
sounds good